src/Tracker/trackerData.tsx

```diff
@@ -674,4 +674,61 @@ export const analyticsEventsMapping = new Map<
       },
     },
   ],
+  [
+    'screen_face_face_video_capture_record_button_click',
+    {
+      eventName: 'FACE_VIDEO_CAPTURE_RECORD_BUTTON_CLICKED',
+      properties: { event_type: 'action' },
+    },
+  ],
+  [
+    'screen_face_face_video_capture_recording_next_click',
+    {
+      eventName: 'FACE_VIDEO_CAPTURE_RECORDING_NEXT_CLICKED',
+      properties: { event_type: 'action' },
+    },
+  ],
+  [
+    'screen_face_face_video_confirmation_play',
+    {
+      eventName: 'FACE_VIDEO_CONFIRMATION_PLAY',
```

Comment:
Actually, given this is an action, I would add the action in the name for consistency with other actions where we explicitly say "clicked"
```suggestion
      eventName: 'FACE_VIDEO_CONFIRMATION_PLAY_CLICKED',
```